Gemfile

```diff
@@ -55,6 +55,8 @@ group :development, :test do
   gem 'debug', platforms: %i[mri mingw x64_mingw]
 
   gem 'rspec-rails', '>= 3.9.0'
+
+  gem 'database_cleaner'
```

Comment:
- [ ] capybara should be added to your gemfile:
```
group :development, :test do
   ...
   **gem 'capybara'**
   ...
 end
```

Reply:
well resolved! ✔️